LICENSE

```diff
@@ -0,0 +1,379 @@
+This software is copyright (c) 2011 by DuckDuckGo, Inc. http://duckduckgo.com,
```

Comment:
Fixed, thanks! I must have copied it from [dist.ini](https://github.com/duckduckgo/community-platform/blob/master/dist.ini). Do you want it fixed there too?